rest/route/agent.go

```diff
@@ -1546,3 +1544,104 @@ func (h *checkRunHandler) Run(ctx context.Context) gimlet.Responder {
 
 	return gimlet.NewJSONResponse(fmt.Sprintf("Successfully created check run for  '%v'", t.Id))
 }
+
+// GET /rest/v2/task/{task_id}/github_dynamic_access_token/{owner}/{repo}
+type createGitHubDynamicAccessToken struct {
+	owner  string
+	repo   string
+	taskID string
+
+	permissions github.InstallationPermissions
+
+	env evergreen.Environment
+}
+
+func makeCreateGitHubDynamicAccessToken(env evergreen.Environment) gimlet.RouteHandler {
+	return &createGitHubDynamicAccessToken{
+		env: env,
+	}
+}
+
+func (g *createGitHubDynamicAccessToken) Factory() gimlet.RouteHandler {
+	return makeCreateGitHubDynamicAccessToken(g.env)
+}
+
+func (g *createGitHubDynamicAccessToken) Parse(ctx context.Context, r *http.Request) error {
```

Comment:
nit: can we please use h instead of g here? I know we use g for the createInstallationToken route but all the other routes use `h`, I'm guessing for "handler"